### DISCUSSION TITLE
Transcribe live stream audio from blackboard? Live stream private links?

### DISCUSSION THREAD

**Discussion by ohmygenie** [deepgram/community#186] [General help] [answered]
Hi, 

Please let me know if this feature is possible, enabling me to transcribe live feed audio from online class discussions such as in blackboard.

Thanks!

**Comment by ohmygenie** [comment]
bump

**Comment by lukeocodes** [comment, marked as answer]
You can send pre-signed links, or links including basic auth. You can also send binary data, too.
